### DISCUSSION TITLE
I want to use the mail service provided by outlook

### DISCUSSION THREAD

**Discussion by moeik** [outline/outline#3669] [Self Hosting]
I tried to put in the relevant information, but he didn't work.
How should I set it up？
Please help me to provide an outlook configuration template, Thanks.

```
# To support sending outgoing transactional emails such as "document updated" or
# "you've been invited" you'll need to provide authentication for an SMTP server
SMTP_HOST=smtp.office365.com
SMTP_PORT=587
SMTP_USERNAME=i@xxx.xx
SMTP_PASSWORD=xxx
SMTP_FROM_EMAIL=i@xxx.xx
SMTP_REPLY_EMAIL=i@xxx.xx
SMTP_TLS_CIPHERS=SSLv2  #How should I set it up here?
SMTP_SECURE=true
```

**Comment by tommoor** [comment]
Hi @moeik it's likely you're the first person to try this. According to various articles `SSLv3` is the value you're looking for, you may also try setting `SMTP_SECURE=false` if that change alone doesn't work.

**Reply by moeik** [reply]
> Hi @moeik it's likely you're the first person to try this. According to various articles `SSLv3` is the value you're looking for, you may also try setting `SMTP_SECURE=false` if that change alone doesn't work.

Sorry....I tried to do this, but he still did not take effect, can you test it yourself?

**Reply by tommoor** [reply]
I cannot test it

**Reply by moeik** [reply]
Well, I might need some advice from other users, thank you.
